### DISCUSSION TITLE
请问官网docs如何本地运行，模板很好用

### DISCUSSION THREAD

**Discussion by kemisstep** [rsuite/rsuite#3315] [Q&A] [answered]
clone代码后
进入docs目录
npm install 一直提示错误
error - ../src/Affix/Affix.tsx:2:0
Module not found: Can't resolve 'prop-types'

![image](https://github.com/rsuite/rsuite/assets/117804089/1841bd58-2eb2-48b8-8e44-e56252e4c936)

请问如何解决？

**Comment by kemisstep** [comment]
npm install --save prop-types

这个安装也是无作用

**Comment by SevenOutman** [comment, marked as answer]
@kemisstep 运行官网 docs 请参考一下步骤，

1. 在项目根目录和 `docs/` 目录中分别运行 `npm install` 安装依赖

```shell
npm i && npm i --prefix docs
```

2. 在 `docs/` 目录中运行 `npm run dev`

```shell
npm run dev --prefix docs
```

3. 访问 http://localhost:3000

---

BTW 我会将此 issue 转换为一个 discussion。

**Reply by kemisstep** [reply]
![image](https://github.com/rsuite/rsuite/assets/117804089/84f98ff1-336f-4fe8-996a-88a0a4f5c0bc)

进度条99%，就是不显示页面

**Reply by SevenOutman** [reply]
可能是页面的构建比较慢，你可以在终端查看 Next.js 输出的日志，确认页面是否已经构建完成。如果仍在构建，继续等待一会；如果已经构建完成却不显示页面，尝试刷新

**Comment by kemisstep** [comment]
这个docs的模板很好看，可以独立出来吗？

**Comment by kemisstep** [comment]
@SevenOutman 可以把docs模板独立出来，单独运行吗？

**Reply by SevenOutman** [reply]
docs 目录的代码并未按照作为模板来复用的场景进行组织，但你可以尝试将其复制出来并修改。
